File: system-test/dotnet/SystemTests/Legacy/Mod09/SmokeTests/System/MyShopSmokeTest.cs
using SystemTests.Legacy.Mod09.Base;
using Optivem.Testing;
using Xunit;

namespace SystemTests.Legacy.Mod09.SmokeTests.System;

public class MyShopSmokeTest : BaseScenarioDslTest
{
    [Theory]
    [ChannelData(ChannelType.UI, ChannelType.API)]
    public async Task ShouldBeAbleToGoToMyShop(Channel channel)
    {
        await Scenario(channel).Assume().MyShop().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod09/smoke/system/MyShopSmokeTest.java
package com.mycompany.myshop.systemtest.legacy.mod09.smoke.system;

import com.mycompany.myshop.systemtest.legacy.mod09.base.BaseScenarioDslTest;
import com.mycompany.myshop.testkit.channel.ChannelType;
import com.optivem.testing.Channel;
import org.junit.jupiter.api.TestTemplate;

class MyShopSmokeTest extends BaseScenarioDslTest {
    @TestTemplate
    @Channel({ChannelType.UI, ChannelType.API})
    void shouldBeAbleToGoToMyShop() {
        scenario.assume().myShop().shouldBeRunning();
    }
}

